### DISCUSSION TITLE
Accessing source code file if download frepple using docker image

### DISCUSSION THREAD

**Discussion by Drayang** [frePPLe/frepple#477] [Q&A]
Hi, i download the frepple using docker image but i am quite new to docker. My question is can I access to any source code file and edit them since I couldn't found any code after downloading the frepple via docker. I am using window OS and docker desktop for the installation.

**Comment by jdetaeye** [comment]
In most cases you don't need to.  You'll inherit from the standard frepple docker container, and add your own addons to add your customizations to it.

If you still need/want to, you can create your own container from scratch - full source code is available for Community Edition, and the dockerfile for the container is in https://github.com/frePPLe/frepple-enterprise/blob/cloud-live/contrib/docker/ubuntu-18.04.dockerfile

**Reply by Drayang** [reply]
Thank you for the reply. This link given is 404 error.

**Reply by Drayang** [reply]
Hi, the source code file i refer to is for example the "djangosettings.py" and "erpconnection" folder. I managed to pull the docker image and launch the Frepple on localhost follow the guide here: https://frepple.org/docs/6.19/installation-guide/docker-container.php

The issue now is I would like to enable the ERP connector and i have to uncomment the command in djangosettings.py. However, since it was just a image, I not really have the community open source code in my PC for me to edit. If i want to change any source code, is that mean I must install Frepple via other method such as the debian package (https://frepple.org/docs/6.19/installation-guide/linux-binaries.php)? I have open another discussion here: https://github.com/frePPLe/frepple/discussions/483

**Comment by jdetaeye** [comment]
Wrong repo...

This one should work https://github.com/frePPLe/frepple/blob/master/contrib/docker/ubuntu-18.04.dockerfile
